src/components/jobs_view.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { ReactNode, useEffect, useState } from 'react';
+import { TfiTrash, TfiPlus, TfiPencil, TfiFilter } from "react-icons/tfi";
+import PouchDB from 'pouchdb'
+import PouchDBUpsert from 'pouchdb-upsert'
+import { Button, ListGroup } from 'react-bootstrap';
+import templatesConfig from '../templates/templates_config'
+import Dropdown from 'react-bootstrap/Dropdown';
+
+PouchDB.plugin(PouchDBUpsert);
+
+interface JobListProps {
+  dbName: string;
+}
+
+const JobList: React.FC<JobListProps> = ({ dbName }) => {
+  const db = new PouchDB(dbName);
+  const [sortedJobs, setSortedJobs] = useState<any[]>([]);
+
+  const retrieveJobs = async () => {
+    try {
+      const result = await db.allDocs({ include_docs: true });
+      //console.log(result.rows.map(row => row));
+      const sortedJobs = result.rows.map(row => row.doc?._id);
```

Comment:
Store whatever will be needed to render the component and call any of the update functions.